### DISCUSSION TITLE
Why are certain templates excluded?

### DISCUSSION THREAD

**Discussion by Kalimon12** [projectdiscovery/nuclei#1337] [Q&A] [answered]
[WRN] Could not load template /root/nuclei-templates/fuzzing/xff-403-bypass.yaml: the template was excluded

**Comment by forgedhallpass** [comment, marked as answer]
This is intended behavior. Certain templates types like `fuzz`, `dos`, `misc` are excluded by default, because they can have potential side effects. In order to execute them, you need to explicitly enable them through the `-itags` or `-include-tags` CLI flags, or enable them via the nuclei configuration file.

Please see https://nuclei.projectdiscovery.io/nuclei/get-started/#template-exclusion
